src/main/java/pom/HomePage.java

```diff
@@ -0,0 +1,54 @@
+package pom;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+
+public class HomePage extends BasePage {
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+
+    String playlistName = "Daria46";
+    By userAvatarIcon = By.cssSelector("img.avatar");
+    By playlistNamefield = By.cssSelector("[name='name']");
+    By firstPlaylist = By.cssSelector(".playlist:nth-child(3)");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```